updater/lib/dependabot/service.rb

```diff
@@ -128,7 +128,8 @@ def capture_exception(error:, job: nil, dependency: nil, dependency_group: nil,
         ErrorAttributes::PACKAGE_MANAGER => job&.package_manager,
         ErrorAttributes::JOB_ID => job&.id,
         ErrorAttributes::DEPENDENCIES => dependency&.name || job&.dependencies,
-        ErrorAttributes::DEPENDENCY_GROUPS => dependency_group&.name || job&.dependency_groups
+        ErrorAttributes::DEPENDENCY_GROUPS => dependency_group&.name || job&.dependency_groups,
+        ErrorAttributes::SECURITY_UPDATE => job&.security_updates_only ? "true" : "false"
```

Comment:
why are we passing in true/false as string values?

Reply:
Valid point @abdulapopoola , As the Error object is sent as JSON object to Sentry, the object is stringyfied to JSON string and sent to Sentry via wrapper. See below

![image](https://github.com/dependabot/dependabot-core/assets/167903774/a3750097-6515-4c33-9f12-440edb17aab1)

Now if i don't use true false as string values . here's the object 

![image](https://github.com/dependabot/dependabot-core/assets/167903774/9ad3467b-6172-4ed5-a2d5-c776bf45aa56)
 
I have a small concern how the true/false filters will be applicable on Sentry side. I haven't seen example of assertion (true/false) in Sentry. Having a string assertion will mitigate that issue.

Reply:
I think Sentry can handle `true/false` values because we actually use boolean filters in Sentry. I think using a string will be confusing.

Reply:
@abdulapopoola , reverted to boolean now.